examples/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 3.10)
+project(wasm-tools-examples)
+
+add_library(wasm-tools-examples STATIC
+	wasm-smith.c
+)
+
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../crates/c-api ${CMAKE_CURRENT_BINARY_DIR}/wasm-tools)
+
+target_compile_options(wasm-tools-examples PRIVATE -Wall -Wextra -Wno-deprecated-declarations)
```

Comment:
This might be the line breaking the CI, does the CI transform cmake projects into msvc projects? A little bit weird but I'll test later